job_templates/snippet/property_parameter-definition.xml.em

```diff
@@ -42,5 +42,18 @@ This tests the robustness to whitespace being within the different paths.</descr
           <defaultValue>@(test_args_default)</defaultValue>
           <trim>false</trim>
         </hudson.model.StringParameterDefinition>
+        <hudson.model.ChoiceParameterDefinition>
+          <name>CI_ENABLE_SANITIZER_TYPE</name>
+          <description>This can be used to enable sanitizer based builds and tests</description>
+          <choices class="java.util.Arrays$ArrayList">
+            <a class="string-array">
+              <string>@enable_sanitizer_type_default</string>
+@{
+sanitizer_types.remove(enable_sanitizer_type_default)
+}@
+              <string>@sanitizer_types[0]</string>
```

Comment:
This single line should be replaced with a loop over the remaining `sanitizer_types`.

Reply:
Removed all sanitizer specific logic now as it is not required anymore.